### PR TITLE
mycli 1.10.0

### PR DESCRIPTION
**Pull request by tsroten**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/mycli.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Mycli < Formula
 
   desc "CLI for MySQL with auto-completion and syntax highlighting"
   homepage "http://mycli.net/"
-  url "https://files.pythonhosted.org/packages/22/e5/91ae1334660623610fa431b068ab027ff0260456ebd8331c8228669322a8/mycli-1.9.0.tar.gz"
-  sha256 "70ccf669db7949fe0f97695c82665e0fb737356f2e1d7f227f4eeda29f3311b2"
+  url "https://pypi.python.org/packages/d3/ba/13b9ca22e77fbe6770c39c6fae912ec2fe95bbbbcc7b1ed287851f04c438/mycli-1.10.0.tar.gz"
```
Comment (neutric) [inline Formula/mycli.rb]: Please keep files.pythonhosted.org.

**Comment by neutric** [comment]
Thanks for your contribution, @tsroten! Would you mind also updating the resources `prompt_toolkit` and `PyMySQL`?

**Comment by neutric** [comment]
Thanks a bunch, @tsroten! 🌻

**Comment by tsroten** [comment]
@neutric Thanks for updating those -- sorry for the huge delay :)

**Comment by neutric** [comment]
@tsroten No problem, I hope it was filled with oodles of fun! :)

**Pull request closed without merging**